application/transaction.go

```diff
@@ -1,11 +1,15 @@
 package application
 
 import (
+	"time"
+
 	"github.com/almighty/almighty-core/log"
 
 	"github.com/pkg/errors"
 )
 
+const databaseTransactionTimeout = 6 * time.Minute
```

Comment:
Shouldn't we move it to configuration? And isn't 6 minutes is too long for the transaction timeout by default?